tests/general/InvalidLinkBearTest.py

```diff
@@ -90,6 +90,7 @@ def test_run(self):
         [httpbin](http://httpbin.org/status/200)
         |http://httpbin.org/status/200|
         <h3>Something http://httpbin.org/status/200</h3>
+        repo=\\"http://httpbin.org/status/200\\"
```

Comment:
this is the test, if that doesn't work this URL will not pass the check because `...200\` is invalid